src/openloco/things/CreateVehicle.cpp

```diff
@@ -50,7 +50,7 @@ namespace openloco::things::vehicle
     static loco_global<uint32_t[32], 0x00525E5E> currencyMultiplicationFactor;
     static loco_global<uint8_t, 0x00525FC5> _525FC5;
     static loco_global<uint32_t, 0x00525FB8> _525FB8;             // total used length of _987C5C
-    static loco_global<thing_id_t[64][1000], 0x0096885C> _96885C; // Likely routing related
+    static loco_global<thing_id_t[1000][64], 0x0096885C> _96885C; // Likely routing related
```

Comment:
Ouch, nice catch.

Reply:
There is still a lot of unknown with this. We need to do the route tab of the vehicle window before going much further with nameing as I think there will be a couple of gotchas with respect to this.